docker/lib/dependabot/docker/update_checker.rb

```diff
@@ -69,8 +69,8 @@ def version_up_to_date?
         # If the tag isn't up-to-date then we can definitely update
         return false if version_tag_up_to_date? == false
 
-        # Otherwise, if the Dockerfile specifies a digest check that that is
-        # up-to-date
+        # Otherwise, if the Dockerfile/docker-compose.yml specifies a digest
```

Comment:
For me, this comment is not clear on what the "that that is up-to-date" is referring to? Is it referring to the `digest` or to the `Dockerfile/docker-compose.yml`?

I suggest a change like so:
"Otherwise, if the Dockerfile/docker-compose.yml specifies a digest, check that the "thing that is checked" is up to date"

Change the "thing that is checked" into either `digest` or `Dockerfile/docker-compose.yml`.